update_workbook.py

```diff
@@ -0,0 +1,249 @@
+import json
+from dataclasses import dataclass
+from enum import auto, Enum
+from pathlib import Path
+from types import MappingProxyType
+from typing import Optional
+
+
+# Configuration Variables
+SYNC_FILE_SUFFIXES = ('.py', '.ipynb')
+WORKBOOK_ROOT_DIR = Path('workbooks')
+SOURCE_REPLACEMENT_MESSAGE = '# Try it yourself:\n'
```

Comment:
They don't just try something out that we demonstrated before.
```suggestion
SOURCE_REPLACEMENT_MESSAGE = '# Insert your solution:\n'
```